src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -172,60 +205,73 @@ def _make_array_shape_and_strides(
     def _add_storage(
         self,
         sdfg: dace.SDFG,
+        symbol_arguments: set[str],
```

Comment:
This is optional, but I would say `symbolic_arguments` would be a bit better from a grammar point of view.

-- Note whenever I argue on this level (grammar) be extremely wary.

Do not implement this suggestion:
You have a lot of argument that you pass around now (essentially the reduction identity is the same).
I would suggest that at some point you collect them all in one object that you then pass around.
Thus if you then notices that you have to add something new (in case of `symbl_arguments`) you do not have to change it everywhere.
In my view this is what encapsulation is.